src/components/modules/dashboard/ui/pages/Dashboard.tsx

```
@@ -51,39 +52,53 @@ export const Dashboard = () => {
 
-        <div className="grid grid-cols-1 md:grid-cols-2 gap-4 lg:gap-6">
-          <div className="md:col-span-2">
-            <EscrowVolumeTrendChart
-              data={volumeTrend || []}
-              isLoading={isLoading}
-            />
-          </div>
+        <Tabs defaultValue="general" className="w-full">
+          <TabsList className="grid w-full grid-cols-3">
+            <TabsTrigger value="general">General</TabsTrigger>
+            <TabsTrigger value="milestone-overview">Milestone Overview</TabsTrigger>
```

ESLint

[error] Replace `Milestone·Overview` with `⏎··············Milestone·Overview⏎············`

(prettier/prettier)